main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1597,22 +1601,45 @@ private static void getExtraOnlineInfo(@NonNull final Geocache cache, final Stri
         final String userToken = parseUserToken(page);
         final Observable<LogEntry> logs = getLogs(userToken, Logs.ALL);
         final Observable<LogEntry> ownLogs = getLogs(userToken, Logs.OWN).cache();
-        final Observable<LogEntry> specialLogs = Settings.isFriendLogsWanted() ?
-                Observable.merge(getLogs(userToken, Logs.FRIENDS), ownLogs) : Observable.empty();
-        final Single<List<LogEntry>> mergedLogs = Single.zip(logs.toList(), specialLogs.toList(),
-                (logEntries, specialLogEntries) -> {
-                    mergeFriendsLogs(logEntries, specialLogEntries);
-                    return logEntries;
-                }).cache();
-        mergedLogs.subscribe(logEntries -> DataStore.saveLogs(cache.getGeocode(), logEntries, true));
+        final Observable<LogEntry> friendLogs = Settings.isFriendLogsWanted() ?
```

Comment:
Also unrelated to this PR, but we should really remove all the boilerplate rxutils constructs here. The main blocking thing (loading the page from Internet) is done blocking anyway, all the data parsing happens in no-time in contrast to that. And the whole observable stuff makes the code almost impossible to read.

Reply:
Since I am not familiar with that stuff, I couldn't do it

Reply:
Since `getLogs` is only used inside GCParser for `getExtraOnlineInfo` I think I can change it to a "simple" list. If it is not observable anymore, then a new LogEntry should be possible to handle. Will check on that.

Reply:
@eddiemuc  Hm - getLogs does a http request, which I guess should stay blocking... 
So I am not sure, how to change that. Any help appreciated

Reply:
I tried to use toList().blockingGet -> that seems to work

Reply:
Maybe I add an marker whether the logs have changed and need to be stored, to avoid unneccesary db-access